dhis-2/dhis-web-api-test/pom.xml

```diff
@@ -209,6 +389,27 @@
       <scope>test</scope>
     </dependency>
 
+    <!--    <dependency>-->
```

Comment:
I would remove it instead of commenting it if we don't need it

Reply:
Done